source/_patterns/01-atoms/svgicon/demo/svgicons.twig

```diff
@@ -0,0 +1,109 @@
+{% extends '@atoms/grid/_grid-1-up.twig' %}
+  {% set container = true %}
+  {% block column_1 %}
+
+    <h1>SVG Icons</h1>
+    <p class="lead">Store your SVG files in the SVG subdirectory to have them made available via the <code>_svgicon.twig</code> partial or using a Sass mixin. SVGs are rendered utilizing their filename as a variable.</p>
+
+    <h2>Using the Sass mixin</h2>
+    <p>The Sass mixin is located in the components scss/_icons-generated.scss file which is generated by the Webpack plugin on build. Use the <code>sprite()</code> mixin on your CSS class with the SVG filename as an argument. This class also needs to have a designated height and width. For your convenience, a CSS class is created for each icon in svg/ with the settings below as a default. These follow the pattern <code>icon--[svg-filename]</code></p>
+    <pre class="language-css mb-3">
+      <code>
+        .svgicon-demo-0 {
+          @include sprite('chevron-right');
+          height: 40px;
+          width: 40px;
+        }
+      </code>
+    </pre>
+      <div class="d-flex justify-content-between mb-3">
+        {% for icon in scssIcons %}
+        <div class="icon--{{ icon }}">
+        </div>
+        {% endfor %}
+      </div>
+
+    <h3>Changing colors with the Sass mixin</h3>
+    <p>To allow an SVG to be colorized, add <code>var:color.fill="#000"</code> to all path elements on the raw SVG. This defaults the color to black, but allows an override when using the mixin. <b class="text-danger">Warning:</b> this does involve altering the raw SVG which will need to be be updated with any revisions to the source file.</p>
```

Comment:
I want a table on this page with "techniques" per row, and "features" as columns. That way we can quickly see "how do I color an SVG that was included only via html class?" or "How do I include a full SVG element and also colorize it?"

Reply:
Added initial implementation in 930b025